contracts/HATVaults.sol

```diff
@@ -280,29 +281,31 @@ contract  HATVaults is Governable, ReentrancyGuard {
     event SetVestingParams(uint256 indexed _pid, uint256 indexed _duration, uint256 indexed _periods);
     event SetHatVestingParams(uint256 indexed _duration, uint256 indexed _periods);
 
-    event ClaimApproved(address indexed _committee,
-                    uint256 indexed _pid,
+    event ApproveClaim(uint256 indexed _pid,
                     address indexed _beneficiary,
                     uint256 _severity,
                     address _tokenLock,
                     ClaimBounty _claimBounty);
 
-    event ClaimSubmitted(uint256 indexed _pid,
+    event SubmitClaim(uint256 indexed _pid,
                             address indexed _beneficiary,
                             uint256 indexed _severity,
                             address _committee);
 
-    event WithdrawRequest(uint256 indexed _pid,
-                        address indexed _beneficiary,
+    event WithdrawRequest(address indexed _beneficiary,
+                        uint256 indexed _pid,
                         uint256 indexed _withdrawEnableTime);
 
     event SetWithdrawSafetyPeriod(uint256 indexed _withdrawPeriod, uint256 indexed _safetyPeriod);
-    
     event SetRewardMultipliers(uint256[24] _rewardMultipliers);
-    
     event SetClaimFee(uint256 _fee);
-
-    event RewardDepositors(uint256 indexed _pid, uint256 indexed _amount);
+    event RewardDepositors(address indexed user, uint256 indexed pid, uint256 indexed amount);
+    event DepositHATReward(address indexed user, uint256 indexed amount);
+    event ClaimReward(address indexed user, uint256 indexed pid);
+    event UpdatePool(uint256 indexed pid, uint256 indexed rewardPerShare);
+    event SetWithdrawRequestParams(uint256 indexed withdrawRequestPendingPeriod, uint256 indexed withdrawRequestEnablePeriod);
+    event DismissClaim(uint256 indexed pid, address indexed sender);
```

Comment:
see earlier comments on adding `msg.sender` to events

Reply:
removed them all